web/client/components/data/identify/IdentifyContainer.jsx

```diff
@@ -144,7 +145,8 @@ export default props => {
                             />
                         </div>
                     </Row>,
-                    <Row className="coordinates-edit-row">
+                    displayByLocalConfig(localConfig, 'mobile', 'Identify', 'showNotifications') &&
```

Comment:
@tonym3nsa I think we need to remove the displayByLocalConfig function using the localConfig configuration already.
localConfig.json has different Indentify configs based on the configuration and currently we have "desktop", "mobile" and "embed" for map.

current mobile identify configuration
```js
{
    "name": "Identify",
    "showIn": ["Settings"],
    "cfg": {
        "showFullscreen": true,
        "position": "bottom",
        "size": 0.5,
        "fluid": true,
        "viewerOptions": {
            "container": "{context.ReactSwipe}"
        }
    }
}
```

expected new configuration 

```diff
{
    "name": "Identify",
    "showIn": ["Settings"],
    "cfg": {
        "showFullscreen": true,
        "position": "bottom",
        "size": 0.5,
+       "disableInfoAlert": true,
+       "disableCoordinatesRow": true,
        "fluid": true,
        "viewerOptions": {
            "container": "{context.ReactSwipe}"
        }
    }
}
```

this properties are mapped automatically as props to the root component of the plugin by the plugin architecture of mapstore.

This mean tha you can use `disableInfoAlert` prop in the component and this will be true only for the mobile config

```diff
- displayByLocalConfig(localConfig, 'mobile', 'Identify', 'showNotifications') &&
+ !disableCoordinatesRow &&
```

then you can pass also disableCoordinatesRow to the Viewer component and apply the same approach to the DefaultViewer component for the `disableInfoAlert`

so please let me know if it's clear the cfg plugin functionality or if I missing some particular corner case for this implementation that must be taken account.